app/code/Magento/Sales/Block/Adminhtml/Order/Create/Form.php

```diff
@@ -182,7 +182,7 @@ public function getOrderDataJson()
                 );
             }
         }
-        if ($this->getStoreId() !== null) {
+        if (!empty($this->getStoreId())) {
```

Comment:
I would recommend simplifying this condition to:
```
if ($this->getStoreId()) {
```

Reply:
I have refactored the if condition